posthog/models/sharing_configuration.py

```diff
@@ -35,6 +35,9 @@ class SharingConfiguration(models.Model):
         unique=True,
     )
 
+    password_required = models.BooleanField(default=False, null=True, blank=True)
+    password = models.CharField(max_length=256, null=True, blank=True, unique=False)
```

Comment:
Yeah this is a big no 😅 Even for this kind of thing storing a plaintext password just looks bad.

Given we never need to _read_ the password, you can use the same hashing logic that the User model has (check out the `make_password` and `check_password` utils from django

Reply:
I think we do want read ability - these are used across different people in an org and shared with clients.

I think it would be annoying if when an organization forgot the password they had to reset it and send it to all stakeholders.